src/styles/createTypography.js

```diff
@@ -21,75 +27,79 @@ export default function createTypography(palette: Object, typography: Object | F
       fontWeightRegular,
       fontWeightMedium,
       display4: {
-        fontSize: 112,
+        fontSize: `${112 / defaultSize}rem`,
         fontWeight: fontWeightLight,
         fontFamily,
         letterSpacing: '-.04em',
-        lineHeight: 1,
+        lineHeight: `${round(128 / 112)}em`,
+        marginLeft: '-.06em',
         color: palette.text.secondary,
       },
       display3: {
-        fontSize: 56,
+        fontSize: `${56 / defaultSize}rem`,
         fontWeight: fontWeightRegular,
         fontFamily,
         letterSpacing: '-.02em',
-        lineHeight: 1.35,
+        lineHeight: `${round(73 / 56)}em`,
+        marginLeft: '-.04em',
         color: palette.text.secondary,
       },
       display2: {
-        fontSize: 45,
+        fontSize: `${45 / defaultSize}rem`,
         fontWeight: fontWeightRegular,
         fontFamily,
-        lineHeight: '48px',
+        lineHeight: `${round(48 / 45)}em`,
+        marginLeft: '-.04em',
         color: palette.text.secondary,
       },
       display1: {
-        fontSize: 34,
+        fontSize: `${34 / defaultSize}rem`,
         fontWeight: fontWeightRegular,
         fontFamily,
-        lineHeight: '40px',
+        lineHeight: `${round(41 / 34)}em`,
+        marginLeft: '-.04em',
```

Comment:
What is this magic number?

Reply:
It helps to keep the first letter starting on the edge. I have Material Component Web is using the same pattern but with a greater negative value. I'm not keen on negative margins, especially on the left/right axis as can cause issues with the body. I have used a small value instead.

Reply:
This causes `<Typography type="display3" align="center"/>` for example to be off-center by .04em. Is there a better way to deal with this?

Reply:
@lewiscowper Perhaps simply not setting `marginLeft` if `align` is `center`? I could prepare a PR with that.

Reply:
I'm wondering whether it's useful to have that margin at all. Isn't that an issue with the font itself? Even if you remove the -margin when align="center", what happens here:
```
<div style={{display: 'flex', flexDirection:'column', flex: 1, justifyContent: 'center'}}>
<MyIcon/>
<Typography type='display4'>
Centered?
</Typography>
</div>
```
You'd have to notice the `-0.04em` offset and add `align='center'` to get rid of it, even if the container itself is centered via flexbox.

Reply:
> I'm wondering whether it's useful to have that margin at all. Isn't that an issue with the font itself?

I have used [material-components-web](https://github.com/material-components/material-components-web/blob/master/packages/mdc-typography/_variables.scss#L37) as inspiration. It could be an issue with the font itself, hard to tell. I have no font contruction experience.

Reply:
It's funny that we don't need a margin-right override, just a margin-left is needed:
I have been playing with the current version. The current version seems to be correctly (n°3) center: https://codesandbox.io/s/6y0ll4m07n. Well, even with a measurement tool, it's hard to tell which version is better centered (n°3 vs n°4). So I would vote for keeping things as they are.